modules/apps/object/object-rest-test/src/testFunctional/tests/FilterCustomObjectsMultipleLevelsRelationshipsWithStringFunctions.testcase

```diff
@@ -77,6 +77,126 @@ definition {
 		}
 	}
 
+	@disable-webdriver = "true"
+	@priority = 4
```

Comment:
I think we need to add property portal.acceptance = "true"; into the tests for priority = 4, could you help check and update this file?

Reply:
All of the tests miss the property portal.acceptance = "true"; I will add it, and this test FP eq 4